experiment_helpers/run_measureFormants_audapter.m

```diff
@@ -134,22 +131,13 @@
         data.bChangeOst = 0; 
 
         %plot amplitude and ost tracking
-        bGoodTrial = check_rmsThresh(data,rmsThresh,h_sub(3));
-    %     figure(h_fig(ctrl))
-    %     subplot(h_sub(3))
-    %     yyaxis left
-    %     plot(data.rms(:,1));
-    % %     ylim([0 0.1]) 
-    %     
-    %     yyaxis right
-    %     plot(data.ost_stat);
-    %     hline(0.01,'k',':');
-    %     
+        bGoodTrial = check_rmsThresh(data,[],h_sub(3), expt.amplcalc);
+
         % clear screen
         delete_exptText(h_fig,h_text)
         clear h_text
 
-        if ~bGoodTrial
+        if ~bGoodTrial && itrial > 9
```

Comment:
run_checkLPC is calling this function with a "quite low amplitude" setting, right? So rather than only applying that low threshold after the 9th trial, I think we want to apply it from the beginning.

Reply:
I disagree. I would like to give the participant several trials to settle into a speaking volume before they see any automatically-generated messages.

Reply:
If the participant starts out speaking so quietly that they are below even the (very generous) pretest threshold, what is the point of waiting 9 trials to tell them to change their volume? Any adjustments the experimenter makes in those trials will not be relevant as soon as they get the instruction to be louder.